src/source/stream.c

```diff
@@ -231,10 +233,18 @@ DataFrameHandle Kvs_streamAddDataFrame(StreamHandle xStreamHandle, DataFrameIn_t
         while (pxListItem != pxListHead)
         {
             pxDataFrameCurrent = containingRecord(pxListItem, DataFrame_t, xDataFrameEntry);
-            if (pxDataFrame->xDataFrameIn.uTimestampMs < pxDataFrameCurrent->xDataFrameIn.uTimestampMs)
+            if (pxDataFrame->xDataFrameIn.uTimestampMs < pxDataFrameCurrent->xDataFrameIn.uTimestampMs ||
+                pxDataFrame->xDataFrameIn.uTimestampMs == pxDataFrameCurrent->xDataFrameIn.uTimestampMs && pxDataFrame->xDataFrameIn.xTrackType == TRACK_VIDEO)
```

Comment:
Does this mean that we only need to worry about `TRACK_VIDEO`? Will similar issue happen on audio?